core/domain/story_jobs_one_off.py

```diff
@@ -94,3 +94,55 @@ def reduce(key, values):
                 sum(ast.literal_eval(v) for v in values))])
         else:
             yield (key, values)
+
+
+class UpdateStoryThumbnailSizeOneOffJob(jobs.BaseMapReduceOneOffJobManager):
```

Comment:
Where are the tests for this job?

Reply:
added 2 test cases, need to add one more test testing the `failing_validation`

Reply:
@seanlip Shouldn't these tests be written using Beam, since this would be run after June release, or is this run separately?

Reply:
I think we're hoping to get this PR in before the July cut, in which case MR jobs would be OK. If it goes in later then Beam jobs would be needed.